Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -440,6 +440,23 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     return .visitChildren
   }
 
+  public override func visit(_ node: AvailabilityVersionRestrictionSyntax) -> SyntaxVisitorContinueKind {
+    if shouldSkip(node) {
+      return .skipChildren
+    }
+    if let unexpected = node.unexpectedBetweenPlatformAndVersion {
```

Comment:
`unexpected` could theoretically be anything if we changed the parser, so just to be sure, I’d suggest to check that `unexpected` really only contains a `>=` operator here:

```swift
unexpected.onlyToken(where: { $0.tokenKind == .binaryOperator(">=") }) != nil
```